### DISCUSSION TITLE
[uTools]通过 uTools 检索简悦在本地的稍后读

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3108] [服务]
> 来自简悦 [Telegram 群](https://t.me/simpreadgroup) 用户 [SettingDust](https://github.com/SettingDust) 👏

# 描述

> 使用 uTools 快速查询简悦在本地的稍后读

https://user-images.githubusercontent.com/81074/143542448-2b66f69b-1b05-4415-844d-4a8e49d4f3ad.mp4

# uTools

> [uTools](http://u.tools/) 是一个 Mac，Windows，Linux 全平台支持且类似 Alfred 的效率工具。

# 功能

> 与 Alfred 一样提供在 Desktop 对简悦稍后读的检索功能，包括：

1. `#keywords` → 检索标签，包括：稍后读和标注的标签

2. `keywords` → 检索任意内容，包括：标题，描述，备注，标签，标注的备注，标签

3. 当打开这个稍后读后会优先打开它在本地的快照（如没有快照的话则直接打开原链接）

# 如何安装（升级）

>  请在 uTools 插件市场搜索 `简悦` 安装即可。（首次安装并设置后，建议重启 uTools）

![image](https://user-images.githubusercontent.com/81074/143804173-5263308c-c9fe-4fd6-96d9-64c0051ccc80.png)

# 配置路径

1. 使用此功能前需要配置本地所在的简悦配置文件，请事先将 `simpread_config.json` 本地路径复制到剪切板

2. 在 uTools 输入 <kdb>src</kdb> 即可呼出设置

![image](https://user-images.githubusercontent.com/81074/143536624-25346910-4b4f-491f-a176-fa4d81337792.png)

3. 然后在其中粘贴，注意要包含 simpread_config.json  e.g.  `/Users/xxxNutstore/SimpRead/simpread_config.json`

4. 当设置完毕后，使用 <kdb>src</kdb> 即可呼出设置可以看到已经设置的路径

# 更多定制化

> 下图的配置是 0.2.2 版的功能（如不是的话，请在应用商店升级）

![image](https://user-images.githubusercontent.com/81074/177230130-16d0e504-9176-4965-9cba-c71608bdfece.png)

- 浏览器

  可指定任意浏览器（需要全路径），适合多浏览器共享一份简悦数据的用户。

- 使用 URL Scheme 打开

  此功能为同步助手 1.1.x 包含的功能，支持通过 URL Scheme 打开本地稍后读，如何配置以使用 [请看这里](https://github.com/Kenshin/simpread/discussions/3864)。

- URL 前缀

  简悦支持 [自建知识库方案](https://github.com/Kenshin/simpread/discussions?discussions_q=label%3A%22self+built%22+)，如果你使用了此方式，可自定义 URL。（需将 URL Scheme 关闭）

# 相关快捷键

> 插件内置了一些关键字作为快捷功能，可根据自身删减。

![image](https://user-images.githubusercontent.com/81074/143547637-ec6b53ef-c706-429f-b04a-ad94f734d80d.png)

# 关联

- [在 Alfred 中检索简悦的稍后读](http://ksria.com/simpread/docs/#/Alfred)

# 引申

- [当简悦变成你的知识库（离线且具有永久链接）](https://github.com/Kenshin/simpread/discussions/2221)

**Comment by ghost** [comment]
生成 `simpread_config.json` 并同步到本地的方法：[自动同步](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%90%8C%E6%AD%A5)

**Reply by Kenshin** [reply]
嗨，谢谢提示。（我帮你整理了下结构）

PS：其实引申段落里面算是说明了这点。
